<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GainChipWidget</class>
 <widget class="QWidget" name="GainChipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="laserEnabledLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Laser Enabled</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="TempPlot" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>781</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="laserGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Laser Controller</string>
   </property>
   <widget class="QLineEdit" name="setCurrentEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="setCurrentLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set Current</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laserEnableButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Laser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setCurrentButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Current</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="tempReadoutBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Temperature</string>
   </property>
   <widget class="QLabel" name="tempReadout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Â°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="laserCurrentBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Laser Current</string>
   </property>
   <widget class="QLabel" name="laserCurrentLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="connectionGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="comComboBox">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="comLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>COM Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comConnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="TECLED">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="deviationBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>120</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Deviation</string>
   </property>
   <widget class="QLabel" name="deviationLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>mK</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="tecEnabledLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TEC Enabled</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="tecGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>TEC Controller</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="setTemperatureEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>25.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="setTemperatureLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set Temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Â°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="PEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>P Coefficient</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ILabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>I Coefficient</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>D Coefficient</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="IEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="DEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setTECButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableTECButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable TEC</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="laserLED">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
